python/tests/test_physical_layout.py

```diff
@@ -33,6 +33,12 @@ def test_vrl_and_lrsh_mismatch():
         _ = dlisio.load('data/chap2/wrong-lrhs.dlis')
     assert "visible record/segment inconsistency" in str(excinfo.value)
 
+@pytest.mark.xfail(strict=True)
```

Comment:
Same comment regarding the `reason` argument. Or turn it from a double fail to a pass? I'm not 100% sure that we should fail on inconsistent attributes if we dont have to, but thats a discussion for later